docs/ig.md

```diff
@@ -157,3 +157,32 @@ test-host                      24598            24577            cat
 ```
 
 Events generated from containers have their container field set, while events which are generated from the host do not.
+
+### Using ig with kubectl debug
+
+Examples of commands:
```

Comment:
Please add a link to the documentation of [kubectl debug](https://kubernetes.io/docs/tasks/debug/debug-cluster/kubectl-node-debug/).